tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "strict": true,
     "target": "es2017",
     "allowJs": true,
-    "esModuleInterop": true
+    "skipLibCheck": true,
+    "esModuleInterop": true,
```

Comment:
Can we also add `"declaration": true` in the `tsconfig.json` and remove the types folder from src.?